src/SprykerEco/Yves/Payone/Plugin/SubFormsCreator/AbstractSubFormsCreator.php

```diff
@@ -21,31 +22,31 @@ abstract class AbstractSubFormsCreator
     /**
      * @return \SprykerEco\Yves\Payone\Plugin\PayoneCreditCardSubFormPlugin
      */
-    protected function createPayoneCreditCardSubFormPlugin()
+    protected function createPayoneCreditCardSubFormPlugin(): PayoneCreditCardSubFormPlugin
```

Comment:
Change the return type hint to `use Spryker\Yves\StepEngine\Dependency\Plugin\Form\SubFormPluginInterface;
`.

Reply:
++